ethcontract/src/transaction/build.rs

```diff
@@ -7,7 +7,7 @@
 use crate::errors::ExecutionError;
 use crate::secret::{Password, PrivateKey};
 use crate::transaction::gas_price::GasPrice;
-use crate::transaction::{Account, TransactionBuilder};
+use crate::transaction::{kms, Account, TransactionBuilder};
```

Comment:
I think this needs to be:

```suggestion
use crate::transaction::{Account, TransactionBuilder};
#[cfg(feature = "aws-kms")]
use crate::transaction::kms;
```